src/isomorphic/React.js

```diff
@@ -112,8 +112,7 @@ if (__DEV__) {
           didWarnPropTypesDeprecated,
           'Accessing PropTypes via the main React package is deprecated,' +
             ' and will be removed in  React v16.0.' +
-            ' Use the prop-types package from npm instead.' +
-            ' Version 15.5.10 provides a drop-in replacement.' +
+            ' Use the latest available 15.* prop-types package from npm instead.' +
```

Comment:
Let's use `v` consistently in the message — either `v15` or we can drop `v` from "React v16".